### ISSUE TITLE
list index out of range

### ISSUE THREAD

**Issue by jschiefner** [ChristopherRogers1991/python-irsend#1]
```python
>>> irsend.list_remotes()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/pi/.local/lib/python2.7/site-packages/py_irsend/irsend.py", line 49, in list_remotes
    remotes = [l.split()[-1] for l in output.splitlines()]
IndexError: list index out of range
```

this occurs when trying to call the `list_remotes()` function aswell as `list_codes()`

**Comment by ChristopherRogers1991** [comment]
Both of those functions run the relevant irsend commands, and save the result in a variable called `output`. Can you try printing the value of `output` and pasting the result here?

**Comment by jschiefner** [comment]
Hey @ChristopherRogers1991 glad you answered :) the `output` variable for list_remotes contains the following string:

```
devinput
fernbedienung
devinput


```
playing around with it a little more gives the following results:

```
>>> print output
devinput
fernbedienung
devinput


>>> remotes = [l.split()[-1] for l in output.splitlines()]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
IndexError: list index out of range
>>> output.splitlines()
['devinput', 'fernbedienung', 'devinput', '']
>>> 'devinput'.split()
['devinput']
>>> 'devinput'.split()[-1]
'devinput'
```

for list_codes the output is the following: (i am printing output here too)
```
>>> irsend.list_codes("fernbedienung")
00000000e0e006f9 KEY_UP
00000000e0e046b9 KEY_RIGHT
00000000e0e0a659 KEY_LEFT
00000000e0e0e01f KEY_VOLUMEUP
00000000e0e0d02f KEY_VOLUMEDOWN
00000000e0e08679 KEY_DOWN
00000000e0e058a7 KEY_MENU
00000000e0e016e9 KEY_ENTER
00000000e0e0f00f KEY_MUTE
00000000e0e0807f KEY_CHANNEL
00000000e0e0d22d KEY_OPTION
00000000e0e0f807 KEY_INFO
00000000e0e0e21d KEY_PLAY
00000000e0e052ad KEY_PAUSE
00000000e0e040bf KEY_POWER


Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/pi/.local/lib/python2.7/site-packages/py_irsend/irsend.py", line 80, in list_codes
    codes = [l.split()[-1] for l in output.splitlines()]
IndexError: list index out of range
>>>
```

**Comment by ChristopherRogers1991** [comment]
It looks like both of those have an empty line at the end, which is causing it to choke:
```
>>> ''.split()[-1]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
IndexError: list index out of range
```

The solution is probably to eliminate the blank lines. It shouldn't be too hard to filter them out. I'll push up a change to a branch.

Out of curiosity, do you have any guesses about what's causing the blank lines? I've used this on a couple different versions of Raspbian, and a couple different versions of Ubuntu, and have not seen this happen.

**Comment by ChristopherRogers1991** [comment]
Alright, I think https://github.com/ChristopherRogers1991/python-irsend/tree/bugfix/index_out_of_range should fix it. @jschiefner, can you test that branch, and let me know if it resolves the issue for you?

**Comment by ChristopherRogers1991** [comment]
https://github.com/ChristopherRogers1991/python-irsend/pull/2 has been merged, which resolves this issue. It has been released in [1.0.1](https://pypi.org/project/py-irsend/1.0.1/), and can be installed with pip.
